src/test/java/org/sagebionetworks/bridge/exporter/integration/TestUserHelper.java

```diff
@@ -117,6 +120,11 @@ public Builder withRoles(Role...roles) {
             return this;
         }
         
+        public Builder withSignUp(SignUp signUp) {
+            this.signUp = signUp;
+            return this;
+        }
```

Comment:
In a subsequent check-in, we'll probably want to consolidate all of this to https://github.com/Sage-Bionetworks/BridgeIntegTestUtils/blob/master/src/main/java/org/sagebionetworks/bridge/user/TestUserHelper.java and share this across all 3 integ tests projects, rather than maintaining 3 different copies.